src/Controls/src/Core/WebView.cs

```diff
@@ -117,13 +117,22 @@ public async Task<string> EvaluateJavaScriptAsync(string script)
 		}
 
 		public void GoBack()
-			=> GoBackRequested?.Invoke(this, EventArgs.Empty);
+		{
+			Handler?.Invoke(nameof(IWebView.GoBack));
+			GoBackRequested?.Invoke(this, EventArgs.Empty);
```

Comment:
Thoughts on hiding everything that's inside "WebView.cs" that's `[EditorBrowsable(EditorBrowsableState.Never)]` and only there for the `IWebViewController` behind explicit interfaces?

Reply:
Yeah i think so, @jsuarezruiz  can you make these changes?

Reply:
But, al this events are already hiddden, no?https://github.com/dotnet/maui/blob/201e60a71ebb95366f9f5c561e0db9fde62a88ca/src/Controls/src/Core/WebView.cs#L190

Reply:
Kind of but we're trying to move away from using `EditorBrowsable`

The whole reason those were there in the past was because we didn't enable IVT but now that we have IVT set there's no need to use `EditorBrowsable`